service/service.go

```diff
@@ -153,9 +153,11 @@ func New(config Config) (*Service, error) {
 				GroupsClaim:    config.Viper.GetString(config.Flag.Service.Installation.Tenant.Kubernetes.API.Auth.Provider.OIDC.GroupsClaim),
 				GroupsPrefix:   config.Viper.GetString(config.Flag.Service.Installation.Tenant.Kubernetes.API.Auth.Provider.OIDC.GroupsPrefix),
 			},
+			SSOPublicKey: config.Viper.GetString(config.Flag.Service.Tenant.SSH.SSOPublicKey),
+
+			DockerhubToken:  config.Viper.GetString(config.Flag.Service.Registry.DockerhubToken),
```

Comment:
Is this something specific to Docker Hub or is this just normal docker credentials? What happens when the registry domain is Quay (or something other than Docker)?

Reply:
It's docker credentials used by `k8s-kubelet.service` to authorize/authenticate against DockerHub API.
When they are missing, we might hit QPS limits.
If registry domain is set to anything other than DockerHub, they are not utilized, but still present on the node fs.

Reply:
It feels like `DockerRegistryCredentials` would have been a better name since it's not Hub specific. But this name is used in the other PRs so it's fine for now.